packages/engine/src/interaction/components/Interacts.ts

```diff
@@ -6,7 +6,8 @@ import { Intersection } from "three";
 export class Interacts extends Component<Interacts> {
   public focusedInteractive: Entity | null
   public focusedRayHit: Intersection | null
-  public subFocusedArray: any[] | null
+  public focusedBoxHit: unknown | null
```

Comment:
probably you should add type `BoxHitResult` with something like `[ Entity, boolean, number?, number? ]`
instead of just `unknown`
/// 
[entityIn, frustum.intersectsBox(aabb), aabb.distanceToPoint(transform.position), index]